back-end/app/Http/Controllers/User/UsersController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Hash;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class UsersController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users = User::where('is_admin', false)->get();
+
+        return response()->json([
+            'data' => $users,
+        ], 200);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+
+        $user = $request->validate([
+            'first_name' => 'required|max:191',
+            'last_name' => 'required|max:191',
+            'email' => 'required|unique:users|max:191',
+            'password' => 'required',
```

Comment:
You don't have password confirmation field in your frontend? Because if it has, you need to validate it as well to have same value with password